test/battle/ability/super_luck.c

```diff
@@ -0,0 +1,22 @@
+#include "global.h"
+#include "test/battle.h"
+
+SINGLE_BATTLE_TEST("Super Luck increases the critical hit ratio by 1 stage")
+{
+    u32 genConfig = 0;
+    for (u32 j = GEN_2; j < GEN_LATEST + 1; j++)
```

Comment:
Why do these for loops start at Gen 2 if Gen 1 configs exist?

Reply:
Hmm, right. I need to make Gen 1 crit mechanic tests. Time to hit the books! xD